### PR TITLE
OKTA-570283 Update Classic archive to Okta Classic Engine

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update "Classic archive" to "Okta Classic Engine" in TOC, and remove references to "archive"
- **Is this PR related to a Monolith release?** No
### Resolves:

* [OKTA-570283](https://oktainc.atlassian.net/browse/OKTA-570283)

### CODE REVIEW

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚢

File: packages/@okta/vuepress-site/docs/guides/archive-overview/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Classic archive overview
-excerpt: The classic archive contains Okta classic-compatible versions of documents that have bene updated to support Okta Identity Engine.
+title: Okta Classic Engine overview
+excerpt: The Okta Classic Engine section contains Okta classic-compatible versions of documents that have bene updated to support Okta Identity Engine.
```
Comment (janethu-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-overview/index.md:3]: ...been updated...

File: packages/@okta/vuepress-site/docs/guides/archive-overview/main/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Classic archive overview
-excerpt: The classic archive contains Okta classic-compatible versions of documents that have bene updated to support Okta Identity Engine.
+title: Okta Classic Engine overview
+excerpt: The Okta Classic Engine section contains Okta classic-compatible versions of documents that have bene updated to support Okta Identity Engine.
```
Comment (janethu-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-overview/main/index.md:3]: ...been updated...

File: packages/@okta/vuepress-site/docs/guides/archive-overview/main/index.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
-## Sign users in archived guides
+## Sign users in guides for Classic Engine
```
Comment (janethu-okta) [inline packages/@okta/vuepress-site/docs/guides/archive-overview/main/index.md:6]: ...for Okta Classic Engine

(So the heading is consistent with the others on that page.)

**Review by janethu-okta — APPROVED** [review]
Approved after edits are implemented. Thanks.

File: packages/@okta/vuepress-site/docs/guides/archive-overview/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/archive-overview/index.md:3]: good eye! thanks for catching that!

**Pull request merged**